guacamole-jetty/src/main/java/dev/crafting/guacamole/Parameters.java

```diff
@@ -34,6 +45,14 @@ private Parameters(Parameters from) {
         password = from.password;
     }
 
+    public static String ConfigFile() {
```

Comment:
Naming convention: `configFile`. Instead of the making it a func, let's initialize the `_configFile` as a static field in static constructor.